### PR TITLE
added the docker file to front end and fixed the package.json for docker

### PR DESCRIPTION
**Pull request by LaxminarayanaV7416**
This PR contains very few changes related to dockerizing the the frontEnd Application:

Here are changes list:
- Added Dockerfile to the frontend
- Fixed the material UI React CSS pack to latest version its been renamed as mui/systems instead of mui/styles (NOTE: This change could impact that some doms might not render, could be a simple fix if we can look up the documentation).

Why did I renamed the mui/systems instead of mui/styles?
- Because the mui/styles was depricated here is the changelog we can refer from https://mui.com/system/styles/basics/

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Create app directory
+WORKDIR /usr/src/app
+
+# Copy dependency definitions
+COPY package.json /usr/src/app
+
+# Install dependecies
+RUN npm set progress=false \
+   && npm config set depth 0 \
+   && npm i install
```
Comment (Anjali0407-git) [inline frontend/Dockerfile:13]: @LaxminarayanaV7416 Seems there is a typo in this command

**Comment by Anjali0407-git** [comment]
> Why did I renamed the mui/systems instead of mui/styles?

It's great that you updated mui/styles. Can you confirm it's not breaking the UI rendering?

**Comment by LaxminarayanaV7416** [comment]
I did check it, With the current state of UI its not breaking, but might break in future we need to put an eye on the functionality when we are testing end to end on console we can see and fix them if we encounter any, But as of now its not breaking

File: frontend/Dockerfile (hunk 1 above)
Comment (LaxminarayanaV7416) [reply frontend/Dockerfile:13]: npm i install is a command, which is bascially similar to npm install or npm i, the twice we added to make sure its runs twice to make sure it didnt miss any packages, You know sometimes due to parallel installation it misses the packages this happens rarely!

**Review by LaxminarayanaV7416 — COMMENTED** [review]
Its for making sure we installed all the packages!

File: frontend/Dockerfile (hunk 1 above)
Comment (Anjali0407-git) [reply frontend/Dockerfile:13]: Thanks @LaxminarayanaV7416 It's working perfectly fine. Can you create Readme file and add Docker Commands into it before merging?

**Pull request merged**